packages/fast-form-generator-react/src/form/form-item.array.tsx

```diff
@@ -85,8 +85,8 @@ class FormItemArray extends React.Component<IFormItemArrayProps & IManagedClasse
     public render(): JSX.Element {
         return (
             <div className={this.props.managedClasses.formItemArray}>
-                <div>
-                    <h3>{this.getLabelText()}</h3>
+                <div className={this.props.managedClasses.formItemArray_header}>
+                    <h3 style={{margin: 0}}>{this.getLabelText()}</h3>
```

Comment:
I think we shouldn't be inlining styles here, either we can target it via the formItemArray_header or create a new class and attach it